stories/Parallax.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Parallax from '../src/Parallax';
+
+const stories = storiesOf('javascript/Parallax', module);
```

Comment:
Can this be `const stories = storiesOf('Javascript|Parallax', module);`

Reply:
Fixed!

@alextrastero

Reply:
Has to be `Javascript|Parallax` sorry :( try to run it locally, you'll see how nice and organize it looks

Reply:
Sorry :smile: I it will fix ASAP

@alextrastero

Reply:
Fixed!

Reply:
u da man! thanks